### DISCUSSION TITLE
[Done] Show total torrents size on group (lable and tracker)

### DISCUSSION THREAD

**Discussion by sabersalv** [jesec/flood#244] [Ideas]
Rutorrent has an option `show label sizes`, It's a very useful information

Solved in https://github.com/jesec/flood/pull/369

**Comment by jesec** [comment]
What do you mean? Can you share a screenshot? 

Flood already has "size/count of label"? 

![image](https://user-images.githubusercontent.com/11585141/113239399-3d5f1000-92dd-11eb-9c21-581456ff9f1e.png)

**Reply by sabersalv** [reply]
Total size of torrents. Give you an idea on how big you've seeded for each tracker or group. Most sites calculate bonus points based on seed size.

This is rutorrent
<img width="292" alt="Screen Shot 2021-04-01 at 11 34 36 AM" src="https://user-images.githubusercontent.com/2525544/113240133-ae52f780-92de-11eb-8182-de0ccd98e0bc.png">

This is Transmission Web Control
<img width="307" alt="Screen Shot 2021-04-01 at 11 34 16 AM" src="https://user-images.githubusercontent.com/2525544/113240138-b27f1500-92de-11eb-909f-79cfda556e24.png">

**Reply by jesec** [reply]
Let me see how can it fit into the design.

**Comment by urwrstkn8mare** [comment]
Hi I know this is marked as done but could the Untagged category also have a file size on it?

![image](https://user-images.githubusercontent.com/42769125/163884316-90a79891-a26c-40f4-bf83-4120aaa0c958.png)

See above it would be useful to see the total size of untagged torrents.
